ecs-cli/modules/cli/regcreds/regcreds_app_test.go

```diff
@@ -36,7 +36,7 @@ func TestGetOrCreateRegistryCredentials_WithCredPair(t *testing.T) {
 	testRegistryName := "examplereg.net"
 	testContainers := []string{"logging", "web"}
 
-	testRegistryCreds := make(map[string]readers.RegistryCredEntry)
+	testRegistryCreds := make(map[string]regcredio.RegistryCredEntry)
```

Comment:
Perhaps consider making all these map literals throughout this test file as well